assets/site.js

```diff
@@ -29,4 +29,19 @@ $(document).ready(function() {
             inputSelector: '#search-input',
             debug: false // Set debug to true if you want to inspect the dropdown
             });
+
+    // Adds a CSS class to a blockquote tag that has a marker like [callout_warn]
+    $("blockquote:contains('[callout_warn]')").addClass("callout-warn");
+    $("blockquote:contains('[callout_warn]')").html($("blockquote:contains('[callout_warn]')").html().replace('[callout_warn]',''));
+    $("blockquote:contains('[callout_success]')").addClass("callout-success");
+    $("blockquote:contains('[callout_success]')").html($("blockquote:contains('[callout_success]')").html().replace('[callout_success]',''));
+    $("blockquote:contains('[callout_danger]')").addClass("callout-danger");
+    $("blockquote:contains('[callout_danger]')").html($("blockquote:contains('[callout_danger]')").html().replace('[callout_danger]',''));
+    $("blockquote:contains('[callout_info]')").addClass("callout-info");
+    $("blockquote:contains('[callout_info]')").html($("blockquote:contains('[callout_info]')").html().replace('[callout_info]',''));
+    $("blockquote:contains('[callout_light]')").addClass("callout-light");
+    $("blockquote:contains('[callout_light]')").html($("blockquote:contains('[callout_light]')").html().replace('[callout_light]',''));
+    $("blockquote:contains('[callout_dark]')").addClass("callout-dark");
+    $("blockquote:contains('[callout_dark]')").html($("blockquote:contains('[callout_dark]')").html().replace('[callout_dark]',''));
```

Comment:
I think it would be simpler and cleaner to just use inline HTML in the form
```markdown
<blockquote class="callout-warn">
Foo bar
<blockquote>
```
... rather than having to match things with JS client-side. If you agree, I have an alternative branch for that part, that I can send as a PR credited in your name...

Reply:
Yeah man, I don't mind, its only a little more typing, I was just trying to make the previous guys idea work, lol.

Reply:
Yeah, but I think extending markdown is a bit of a silly thing to do when we can just inline html. But OK, I'll submit my version, then!

Reply:
👍